src/test/java/nl/ordina/jobcrawler/utils/SkillFactory.java

```diff
@@ -0,0 +1,25 @@
+package nl.ordina.jobcrawler.utils;
+
+import nl.ordina.jobcrawler.model.Skill;
+import nl.ordina.jobcrawler.model.Vacancy;
+
+import java.util.HashSet;
+import java.util.UUID;
+
+public class SkillFactory {
+    public static Skill create(String title) {
+        return Skill.builder()
+                .id(UUID.randomUUID())
+                .name(title)
+                .vacancies(new HashSet<Vacancy>())
+                .build();
+    }
+
+    public static Skill create(UUID id, String title) {
+        return Skill.builder()
```

Comment:
if id is available then i think the method name should be update rather than create.

Reply:
That is correct, I needed this method to check against an update endpoint. The method is no longer needed however and the checking can be done in a different way, so it is delete in ec4cc835178e664ca7ef770f8a8b8bceee710592